pymeasure/adapters/adapter.py

```diff
@@ -32,7 +32,12 @@ class Adapter(object):
     techniques.
```

Comment:
Can you please add some documentation about how this feature can be used, e.g. as an additional section in https://pymeasure.readthedocs.io/en/latest/dev/adding_instruments.html#writing-properties?

Reply:
I can hopefully assist with the rst stuff if you need help.

Reply:
I will write some further tests and a first draft for documentation.

Reply:
One pattern for flexible processors that might not be obvious to end-users is to use regexes for matching (can even be several ones), and if none match to pass the string through unmodified. That would be good to include in the docs.